config/recipes/beats/README.md

```diff
@@ -0,0 +1,39 @@
+# Beats Configuration Library
+
+This directory contains yaml manifests with example configurations for Beats. These manifests are self-contained and work out-of-the-box on any non-secured Kubernetes cluster. All of them contain three-node Elasticsearch cluster and single Kibana instance. All Beat configurations set up Kibana dashboards if they are available for a given Beat and all required RBAC resources. 
```

Comment:
Should we add a note that none of these set resource requests/limits and they may want to set these in a production deployment?

Reply:
> none of these set resource requests/limits

should we set default reasonable ones in the examples directly?

Reply:
> > none of these set resource requests/limits
> 
> should we set default reasonable ones in the examples directly?

For Beats, I'd say our defaults are already reasonable. I think we shouldn't overload those examples.

Reply:
Ah I wondered because even on a cluster with very little traffic packetbeat was using up a lot of its CPU, but I'm good with not mentioning it